source/backend/api/Areas/Reports/Mapping/Lease/LeaseMap.cs

```diff
@@ -11,19 +11,19 @@ public class LeaseMap : IRegister
     {
         public void Register(TypeAdapterConfig config)
         {
-            config.NewConfig<(Entity.PimsLeasePeriod period, Entity.PimsLease lease, Entity.PimsPropertyLease property, Entity.PimsLeaseTenant tenant), Model.LeaseModel>()
+            config.NewConfig<(Entity.PimsLeasePeriod period, Entity.PimsLease lease, Entity.PimsPropertyLease property, Entity.PimsLeaseTenant tenant, DateTime? expiryDate), Model.LeaseModel>()
                 .AfterMapping((src, dest) =>
                 {
                     MapLease(src, dest);
                 });
         }
 
-        private static void MapLease((Entity.PimsLeasePeriod period, Entity.PimsLease lease, Entity.PimsPropertyLease property, Entity.PimsLeaseTenant tenant) src, Model.LeaseModel dest)
+        private static void MapLease((Entity.PimsLeasePeriod period, Entity.PimsLease lease, Entity.PimsPropertyLease property, Entity.PimsLeaseTenant tenant, DateTime? expiryDate) src, Model.LeaseModel dest)
         {
             dest.LFileNo = src.lease.LFileNo;
             dest.MotiRegion = src.lease.RegionCodeNavigation?.RegionName;
             dest.StartDate = src.lease.OrigStartDate.FilterSqlMinDate().ToNullableDateOnly();
-            dest.EndDate = src.lease.OrigExpiryDate?.FilterSqlMinDate().ToNullableDateOnly();
+            dest.EndDate = src.expiryDate.ToNullableDateOnly();
```

Comment:
Do we need to filter the sql min date here? previously this would cause exceptions converting from an sql min date to a date time.